web/wp-content/themes/borealis-2021/single-research-blogs.php

```diff
@@ -10,7 +10,9 @@
 
 get_header();
 $hero_image_url = get_the_post_thumbnail_url($post->ID, 'full');
-$post   = get_post($id);
+$title = get_the_title($post->ID);
+$publication_date = get_post_meta($post->ID, 'publication_date', true);
+$authors = get_post_meta($post->ID, 'authors', true);
 $url = $post->guid;
```

Comment:
Can you change this to [get_permalink](https://developer.wordpress.org/reference/functions/get_permalink/) using the post ID? It just looks nicer as a url. The guid will set the post type and ID as params